dev/EnvironmentManager/API/Microsoft.Windows.System.EnvironmentManager.cpp

```diff
@@ -16,20 +17,23 @@ namespace winrt::Microsoft::Windows::System::implementation
 
     Microsoft::Windows::System::EnvironmentManager EnvironmentManager::GetForProcess()
     {
+        EnvironmentManagerInsights::StringTelemetryEvent(L"Making EM for process");
         Microsoft::Windows::System::EnvironmentManager environmentManager{ nullptr };
         environmentManager = winrt::make<implementation::EnvironmentManager>(Scope::Process);
         return environmentManager;
     }
 
     Microsoft::Windows::System::EnvironmentManager EnvironmentManager::GetForUser()
     {
+        EnvironmentManagerInsights::StringTelemetryEvent(L"Making EM for User");
         Microsoft::Windows::System::EnvironmentManager environmentManager{ nullptr };
```

Comment:
NIT: remove nullptr?

Does EnvironmentManager have a void ctor? If so then nullptr can be removed.
(if it doesn't but ctor(ptr=null) is legal then consider adding a void ctor :-)

Reply:
Yep.  nullptr constructor is made by default.  In fact, this has to be done in this case.

https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/author-apis#if-youre-authoring-a-runtime-class-in-a-windows-runtime-component

Whether you have IDL-declared constructor(s) or not, a constructor overload that takes std::nullptr_t is generated on your projected type. **Calling the std::nullptr_t constructor is the first of two steps in consuming the runtime class from the same compilation unit.** For more details, and a code example, see Consume APIs with C++/WinRT.

Reply:
In fact.  If you try anything else the compiler will get stuck in a loop.